File: 프로그래머스/lv0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.js
function solution(my_string) {
    var sum = 0;
    var array = my_string.split("");
    for(i = 0; i < array.length; i++) {
        if(parseInt(array[i]) == array[i]/1) {
            sum = sum + parseInt(array[i])
        }
    } return sum;
}

File: Java/프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.java
class Solution {
    public int solution(String my_string) {
        int count = 0;
        String[] result = my_string.replaceAll("[^0-9]", "").split("");
        
        for(String str: result) {
            count = count + Integer.parseInt(str);
        }
        
        return count;
    }
}